en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

```diff
@@ -143,7 +143,7 @@ Follow the instructions below to invoke the previously created API with the gene
 
 __Congratulations!__ You have successfully created your first API, subscribed to it through an OAuth 2.0 application, obtained an access token for testing, and invoked your API with Choreo Connect.
 
-## See also
+## See Also
```

Comment:
Revert this one. It should ideally be sentence case for all subheadings.

Reply:
Will do. Thanks